plugins/actionlog/joomla/joomla.php

```diff
@@ -1175,4 +1171,25 @@ public function onJoomlaAfterUpdate($oldVersion = null)
 		);
 		$this->addLog(array($message), 'PLG_ACTIONLOG_JOOMLA_USER_UPDATE', $context, $user->id);
 	}
+
+	/**
+	 * Does return the action log params for the given context.
+	 *
+	 * @param   string  $context  The context of the content
```

Comment:
```suggestion
	 * @param   string  $context  The context of the action log
```

Not 100% sure but I think this is what you meant